contracts/gateway.yaml

```diff
@@ -1822,8 +1823,122 @@ paths:
                 $ref: '#/components/schemas/ErrorResponse'
             application/xml:
               schema:
-                $ref: '#/components/schemas/ErrorResponse'                
-  
+                $ref: '#/components/schemas/ErrorResponse'
+  /v0.5/patient/share:
+    post:
+      tags:
+        - patient
+        - hip facing
+      parameters:
+        - $ref: "#/components/parameters/authorization"
+        - $ref: "#/components/parameters/X-HIP-ID"
+      summary: Patient details on Share by user
+      description: >
+        API to submit the patient details on QR code scan.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PatientShareRequest'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/PatientShareRequest'
+      responses:
+        '202':
+          description: accepted
+        '400':
+          description: >
+            **Causes:**
+              * Token is not provided
```

Comment:
This should be Invalid Request Details instead of Token is not provided

Reply:
done